lib/pharos/resources/calico/typha/deployment.yml

```diff
@@ -0,0 +1,77 @@
+# This manifest creates a Deployment of Typha to back the above service.
+
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: calico-typha
```

Comment:
Typha files are not used?

Reply:
Not sure if I like to keep "dead config" here. Could we add these when we actually use them?

Reply:
@jnummelin wdyt?

Reply:
I agree, we should add them only if they're used.

Reply:
They're part of the YAML installation package and referenced by the `daemonset`, just disabled by default... I imagine that we'll want to have optional support for typha eventually, it's recommended for use on clusters >50 nodes.

I don't see how it would hurt to hang onto them.